charts/golang/ci/with-sa-values.yaml

```diff
@@ -0,0 +1,3 @@
+serviceAccount:
+  create: true
```

Comment:
Nit: maybe this should be `enabled: true` to match other similar uses in this chart?

Reply:
`create: true` is a common pattern for service accounts, so we can have the following scenarios:

- `create: false` + `name: ""` = use `default` SA.
- `create: false` + `name: "foo-sa"` = use `foo-sa` SA.
- `create: true` + `name: ""` = Create and use `service-name-golang` SA.
- `create: true` + `name: "my-service"` = Create and use `my-service` SA.